### PR TITLE
Add sentieon readwriter

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/sentieon/readwriter/meta.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        Our software improves upon BWA, STAR, Minimap2, GATK, HaplotypeCaller, Mutect, and Mutect2 based pipelines and is deployable on any generic-CPU-based computing system.
+      homepage: https://www.sentieon.com/
+      documentation: https://www.sentieon.com/
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information.
+        e.g. [ id:'test', single_end:false ]
+  - meta:
```
Comment (asp8200) [inline modules/nf-core/sentieon/readwriter/meta.yml:21]:
```suggestion
  - meta2:
```
Do you know the order in which these fields should be listed? I got the impression that it is standard to list them in the same order in the meta.yml input-section as the listen in the input-section in the main.nf.

File: modules/nf-core/sentieon/readwriter/meta.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      type: map
+      description: |
+        Groovy Map containing reference information.
+        e.g. [ id:'test' ]
+  - meta:
```
Comment (asp8200) [inline modules/nf-core/sentieon/readwriter/meta.yml:26]:
```suggestion
  - meta3:
```

**Review by asp8200 — APPROVED** [review]
Just some commit for the meta.yml. Otherwise perfect

File: modules/nf-core/sentieon/readwriter/meta.yml (hunk 1 above)
Comment (ramprasadn) [reply modules/nf-core/sentieon/readwriter/meta.yml:21]: I have seen the parameters listed in random order in some modules, so I don't think there is one. Or even if there is, I suppose its not strictly followed. 😄

**Pull request merged**